aip/cloud/new.md

```diff
@@ -0,0 +1,123 @@
+even have policies applied to it (like an authorization policy). Unlike other
+resources, a shared policy can be _bound_ to one or more resources, and applies
+to those resources and any descendents. An example of a shared policy is
+billing accounts which are associated with resouces that bill to them.
+
+## Standard Methods
+
+For most policy types, it makes sense to incorporate the policy name in to the
+method name, for example `GetBillingPolicy` for a Billing policy. The
+descriptions below omit the specific policy type for brevity.
+
+### Dedicated Policies
+
+Dedicated polices are the simpler of the two to implement, with a smaller API
+scope. A common issue is that since they are bound to a single resource, the
+permissions to modify the policy get conflated with permissions to modify the
+resource.
+
+- **SetPolicy** - Sets a dedicated policy for a resource
+- **GetPolicy** - Retrieves the dedicated policy for a resource
+- **DeletePolicy** - Deletes the dedicated policy for a resource
+- **LookupEffectivePolicy** - Retrieves the effective policy for a resource,
+  which is the combination of policies on the resource and any ancestors
+
+### Shared Policies
+
+Shared policies have a more complex API scope, but are also substantially more
+powerful. They permit users to have very fine-grained permissions and delegated
+policy roles. They also allow a common policy definition (which might be quite
+complex) to be bound to numerous resoures.
+
+- **CreatePolicy** - Creates a shared policy resource
+- **GetPolicy** - Retrieves a shared policy resource
+- **UpdatePolicy** - Updates a shared policy resource
+- **DeletePolicy** - Deletes a shared policy resource
+- **ListPolicies** - Lists shared policy resources (typically for a given
+  `parent`)
+- **LookupEffectivePolicy** - Retrieves the effective policy for a given
+  `resource`, which is the combination of the polices bound to the resource and
+  any ancestors
+- **CreateBinding** - Creates a binding between a shared policy resource and a
+  specified resource
+- **GetBinding** - Retrieves a binding between a shared policy resource and a
+  resource
+- **DeleteBinding** - Deletes a binding between a shared policy resource and a
+  resource
+- **ListBindings** - Lists bindings between shared policy resources and other
+  resources. This method **must** accept either a reference to the shared
+  policy resource or the target resource (typically via the `filter` field),
+  and list all appropriate bindings.
+
+### Permissions
+
+There **should** be a separate permission for each policy method, per standard
+practice.
+
+Bindings are a bit special in that they connect two resources. Conceptually,
+it's easiest to think of a binding as two half-bindings, one connected to the
+Policy and one connected to the Resource. Each half-binding has it's own
+permission to create/delete. So to create a binding between a Policy and a
```

Comment:
Therefore, creating a binding between a policy and  a resource requires two permissions; however, breaking a binding...